networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/gossip/topics/AttesterSlashingTopicHandlerTest.java

```diff
@@ -60,6 +66,7 @@ public void handleMessage_validSlashing() {
     Bytes serialized = gossipEncoding.encode(slashing);
     final ValidationResult result = topicHandler.handleMessage(serialized);
     assertThat(result).isEqualTo(ValidationResult.Valid);
+    verify(consumer).forward(slashing);
```

Comment:
We should also verify this isn't called in the other cases :D

Reply:
Great point. I've added the checks.